data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -37,8 +37,16 @@ spec:
             - |
               echo 'Installing bridge and tuning CNIs'
               cni_mount_dir=/opt/cni/bin
-              cp --remove-destination /usr/src/github.com/containernetworking/plugins/bin/bridge ${cni_mount_dir}/cnv-bridge || exit 1
-              cp --remove-destination /usr/src/github.com/containernetworking/plugins/bin/tuning ${cni_mount_dir}/cnv-tuning || exit 1
+              sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
+              cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge || exit 1
+              cp --remove-destination ${sourcebinpath}/tuning ${cni_mount_dir}/cnv-tuning || exit 1
+
+              echo 'Checking bridge and tuning CNIs deployment on node'
+              printf -v bridgechecksum "%s" "$(<$sourcebinpath/bridge.checksum)"
+              printf -v tuningchecksum "%s" "$(<$sourcebinpath/tuning.checksum)"
+              printf "%s %s" "${bridgechecksum% *}" "${cni_mount_dir}/cnv-bridge" | sha256sum --check || exit 1
+              printf "%s %s" "${tuningchecksum% *}" "${cni_mount_dir}/cnv-tuning" | sha256sum --check || exit 1
```

Comment:
wdyt about 
`diff <(cat SHA_FILE_OF_INNER_FILE) <(sha256sum INNER_FILE | cut -f 1 -d" ")`

where SHA_FILE_OF_INNER_FILE is created the the bump file using
`sha256sum OUTER_FILE | cut -f 1 -d" "`

Reply:
Current solution uses only builtin bash functions, I prefer it. Your proposal requires `diff` and  `cur` in addition to the `sha256sum`.

Reply:
in case `registry.access.redhat.com/ubi8/ubi-minimal` have them already, (as they are pretty basic and common i think),
i think its more readable and straightforward

Reply:
I do not think it is more readable. You are more familiar with these commands than with `printf`.

Reply:
i find it simpler and believe not just me are more comfortable with cut / cat / diff than `printf -v`
and its also one line instead of two

matter of test, up to the reviewers / author to decide

Reply:
I would vote to use `printf` in this case. it's indeed a matter of taste as all soaps do the dishes properly.